include/ccf/pal/attestation.h

```diff
@@ -2,6 +2,9 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
+#include "ccf/crypto/ecdsa.h"
+#include "ccf/crypto/pem.h"
+#include "ccf/crypto/verifier.h"
```

Comment:
Now that this isn't guarded by `#if !defined(INSIDE_ENCLAVE) || defined(VIRTUAL_ENCLAVE)`, I had to move `ecdsa.h` to the public headers, hence the other changes in this PR.